CHANGELOG.md

```diff
@@ -278,6 +278,14 @@ This update brings the below changes over the previous release:
 ### Breaking Changes
 - Modified column encryption key store provider registrations to give built-in system providers precedence over providers registered on connection and command instances. [#1101](https://github.com/dotnet/SqlClient/pull/1101)
 
+## [Stable Release 2.1.5] - 2022-08-xx
+
+### Fixed
+
+- Added CommandText length validation when using stored procedure command types. [#1726](https://github.com/dotnet/SqlClient/pull/1726)
+- Fixed Kerberos authentication failure when using .NET 6. [#1727](https://github.com/dotnet/SqlClient/pull/1727)
+- Removed union overlay design and use reflection in `SqlTypeWorkarounds`. [#1729](https://github.com/dotnet/SqlClient/pull/1729)
+
```

Comment:
NIT: SqlBinary as rowversion was not an issue on this version correct?

Reply:
Correct